README.md

```diff
@@ -478,6 +478,9 @@ Or as a param in `synchronize` method:
 Photo.synchronize(transaction_per_page: true)
 ```
 
+## Model Aliasing
+By default `synced` gem will look for a class in Core with the same name as it was defined in. If you want to name your class differently or place it in a namespace, provide the actual class name with the `model_name` option. like so: `synced model_name: "Booking"`.
```

Comment:
I'd rather use BookingSync instead of core

Reply:
Also in theory users don't need to know names of models. I'd rather say it's name of endpoint singularized and camelcased. Maybe it's better to use `endpoint` key instead of `model_name`?